Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -94,6 +105,16 @@ void HTMLLinkElement::resource_did_load()
 {
     VERIFY(resource());
 
+    if (m_relationship & Relationship::Stylesheet) {
+        resource_did_load_stylesheet();
+    }
```

Comment:
Was this function previously only called with style sheets? Because now this changes the behavior for the not `m_relationship & Relationship::Stylesheet` case.

Reply:
If this is the case a `VERIFY(stylesheet | icon)` can be nice to indicate / ensure this.

Reply:
In HTMLLinkElement::inserted there are only calls on `set_resource` on 2 conditions:
- In case `Relationship::Stylesheet` was set (there is even a check if there wasn't an alternate keyword)
- The new case where `Relationship::Icon` was set

The confusing part is I don't know to handle yet if both the stylesheet and icon keyword has been set

I'll add the `VERIFY`